epsagon/events/dbapi.py

```diff
@@ -67,30 +67,39 @@ def __init__(
         super(DBAPIEvent, self).__init__(start_time)
 
         self.event_id = 'dbapi-{}'.format(str(uuid4()))
-        dsn = parse_dsn(connection.dsn)
-        self.resource['name'] = dsn['dbname']
 
-        query = cursor.query
+        if hasattr(connection, 'dsn'):
```

Comment:
Please add docs what db module is treated (pg/mysql)

Reply:
👍